starter-code/index.html

```diff
@@ -1,31 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="/Users/Claudz/code/Week1/lab-css-instagram-clone/starter-code/css/main.css">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+  <div class="header"></div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+
+
+  
+
+  <div class="container">
+      <img src="./images/brand.png" alt="instagram" class="insta">
+  <h2>Capture and Share the World's Moments</h2>
+
+  <h3>Instagram is a fast, beautiful and fun way to share your life with friends and family.</h3>
 
   Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
```

Comment:
Surround this text in `<p> </p>` paragraph tags, do not leave text without tags.